_tests/integration/android_test.go

```diff
@@ -2,76 +2,73 @@ package integration
 
 import (
 	"fmt"
-	"path/filepath"
-	"strings"
 	"testing"
 
 	"github.com/bitrise-io/bitrise-init/_tests/integration/helper"
 	"github.com/bitrise-io/bitrise-init/models"
-	"github.com/bitrise-io/bitrise-init/output"
-	"github.com/bitrise-io/bitrise-init/scanner"
 	"github.com/bitrise-io/bitrise-init/steps"
-	"github.com/bitrise-io/go-utils/fileutil"
-	"github.com/stretchr/testify/require"
 )
 
 func TestAndroid(t *testing.T) {
 	var testCases = []helper.TestCase{
 		{
-			"sample-apps-android-sdk22",
```

Comment:
The list of test apps has been consolidated:
- `sample-apps-android-sdk22-subdir` repo is archived and the related test case removed. The subdir test case is covered by `bitrise-init-android-test-apps` .
- `android-non-executable-gradlew` test case removed as we removed extra warnings when the android repo has no gradle wrapper script